nomenclature/__init__.py

```diff
@@ -1,4 +1,7 @@
 import logging
+from pathlib import Path
+from setuptools_scm import get_version
+from importlib.metadata import version
```

Comment:
If we require `importlib.metadata` we should add the fact that this package requires python 3.8 or higher to setup.cfg and the README.md